lib/services/storageManagement2/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "azure-arm-storage",
   "author": "Microsoft Corporation",
   "description": "StorageManagementClient Library with typescript type definitions for node",
-  "version": "6.3.0",
+  "version": "6.4.0",
```

Comment:
This is actually a major version bump because a property was changed from `useSubDomain` to `useSubDomainName`.

Reply:
@daschult Updated and regenerated